modules/entity_version_history/tests/FunctionalJavascript/HistoryTabSettingsTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
```

Comment:
Let's put the tests inside a `src` folder like the parent module. We should aim to this everywhere, and with this occasion, also do it please in the rest of the modules of this project.